pkg/integrations/cloudwatch_exporter/cloudwatch_exporter.go

```diff
@@ -0,0 +1,209 @@
+package cloudwatch_exporter
+
+import (
+	"context"
+	"fmt"
+	"github.com/grafana/agent/pkg/integrations"
+	"time"
+
+	"github.com/go-kit/log"
+	yace "github.com/nerdswords/yet-another-cloudwatch-exporter/pkg"
+	yaceConf "github.com/nerdswords/yet-another-cloudwatch-exporter/pkg/config"
+	yaceLog "github.com/nerdswords/yet-another-cloudwatch-exporter/pkg/logger"
+	yaceModel "github.com/nerdswords/yet-another-cloudwatch-exporter/pkg/model"
+	yaceSess "github.com/nerdswords/yet-another-cloudwatch-exporter/pkg/session"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	integrations.RegisterIntegration(&Config{})
```

Comment:
Don't forget to include a v2 mapping here so the integration can be configured multiple times with vnext.

`integrations_v2.RegisterLegacy(&Config{}, integrations_v2.TypeMultiplex, metricsutils.NewNamedShim("cloudwatch_exporter"))`